satellite/satellitedb/overlaycache.go

```diff
@@ -824,6 +836,11 @@ func (cache *overlaycache) UpdateNodeInfo(ctx context.Context, nodeID storj.Node
 func (cache *overlaycache) UpdateUptime(ctx context.Context, nodeID storj.NodeID, isUp bool, lambda, weight, uptimeDQ float64) (stats *overlay.NodeStats, err error) {
 	defer mon.Task()(&ctx)(&err)
 
+	stats = cache.updateChecker.GetNodeStats(nodeID, isUp)
```

Comment:
what if this races with some other update uptime?

Reply:
See comment above.